packages/avatar-group/src/vaadin-avatar-group.js

```diff
@@ -445,7 +525,7 @@ class AvatarGroup extends ResizeMixin(ElementMixin(ThemableMixin(PolymerElement)
       }
     }
     // Override generated tooltip text
-    this.__overflowTextGenerator = () => result.join('\n');
+    tooltip.generator = () => result.join('\n');
```

Comment:
Since the generator does return a static text, is there any benefit to using the generator over just setting the text directly?

Reply:
Good point, done.